src/org/usfirst/frc/team2335/robot/commands/ShootGroup.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootGroup extends CommandGroup {
+    
+    public ShootGroup()
+    {
+    	DriverStation.reportWarning("Running", true);
```

Comment:
Definitely go over it in a meeting, then, and afterwards (or during the meeting!) add that knowledge to new codebase-wide documentation.